phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixMetricsIT.java

```diff
@@ -487,7 +495,7 @@ public void testMetricsForUpsert() throws Exception {
             String t = entry.getKey();
             assertEquals("Table names didn't match!", tableName, t);
             Map<MetricType, Long> p = entry.getValue();
-            assertEquals("There should have been sixteen metrics", 16, p.size());
+            assertEquals("There should have been sixteen metrics", 22, p.size());
```

Comment:
either remove the text, or update the text
```suggestion
            assertEquals("There should have been 22 metrics", 22, p.size());
```